pkg/lib/time/time.go

```diff
@@ -33,6 +33,10 @@ func MillisecsStr(t time.Time) string {
 	return nanos[0:3]
 }
 
+func SecondStr(t time.Time) string {
```

Comment:
At the moment, no. The reason I added it is that I needed it at some point. Thought of not removing it when I no longer needed it because it fits the theme nicely with the others (`MillisecsStr` and `MicrosecsStr`). What do you think?

Reply:
I think removing it is ok, since padding to 9 places doesn't make as much sense for seconds (since for seconds, 9 would be arbitrary)

Reply:
I went ahead and removed it just to save you the effort of coming back to this branch